front/lib/actions/mcp_internal_actions/servers/slack.ts

```diff
@@ -116,9 +109,7 @@ const _getPublicChannels = async ({
 
 const getCachedPublicChannels = cacheWithRedis(
   _getPublicChannels,
-  ({ mcpServerId }: GetPublicChannelsArgs) => {
-    return `public-channels-${mcpServerId}`;
-  },
+  ({ mcpServerId }: GetPublicChannelsArgs) => mcpServerId,
```

Comment:
@Fraggle FYI cacheWithRedis already puts a prefix based on the method name, e.g. `cacheWithRedis-_getPublicChannels-ims_4aO6liIbrN`.